### DISCUSSION TITLE
Should we simplify creating standard stable blueprints

### DISCUSSION THREAD

**Discussion by viveksinghggits** [kanisterio/kanister#1486] [Ideas]
I was just thinking if it would be a good idea to be able to create blueprint using below command 

```
kanctl create blueprint --app mysql
```
and this is going to create the blueprint that we have here https://github.com/kanisterio/kanister/blob/master/examples/mysql/mysql-blueprint.yaml

The other way is to open the GitHub repo, go to the specific directory, click on Raw button, copy the link and then run below command
```
k create -f https://github.com/kanisterio/kanister/blob/master/examples/mysql/mysql-blueprint.yaml -n kanister
```
If not to users, I think this is going to be helpful to the developers at least.

**Comment by ihcsim** [comment]
It will be nice if the (verified) blueprints can be stored in some public registries like artifacthub.io. Then we can just use common tools like `kubectl`, `helm`, `kustomize` etc. to deploy them.

**Comment by viveksinghggits** [comment]
I was thinking even if they are just there in the kanister repo, and we already know that people use kanctl so  maybe just use kanctl.
